docs/src/static-pages/logo.mdx

```diff
@@ -6,6 +6,8 @@ category: guidelines
 
 ## Table of Contents
 
+## Introduction
```

Comment:
As per my main comment in the PR this was added to fix an issue with the intro text not being displayed when ToC was being used